test/module/shared_model/bindings/builder-test.py

```diff
@@ -0,0 +1,89 @@
+import iroha
```

Comment:
Is there possible that Iroha lib failed on initialization? If yes, this cases handled by the test?

Reply:
Then it'll be exception with smth like "sorry cannot find this stuff" and tests fails as long as it doesn't be handled